content/docs/integrations/stackoverflow/index.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Stackoverflow Plugin
+publishedDate: '2022-06-09T10:00:00.0Z'
+description: How to enable the Stackoverflow plugin
+
+humanName: Stackoverflow Plugin
+logoImage: '../../../assets/logos/stackoverflow/so-logo.png'
+integrationType: OSS plugin
+---
+
+## Introduction
+
+Roadie provides integration to display public and private Stackoverflow questions and answers on the homepage. To configure this plugin for a private Stackoverflow Teams subscription, you need to be able to generate an API key for Stackoverflow. To have the possibility to generate an API key, you will need to have at least 'Basic' level subscription to Stackoverflow Teams.
+
+## Authentication
+
+The Stackoverflow API uses token-based authentication so in order to retrieve results you will need it. Follow the steps below in order to set it.
+
+### Step 1: Generate an API token
+
+To generate an API token, follow the instruction outlined in [the official documentation for Stackoverflow teams](https://stackoverflow.help/en/articles/4385859-stack-overflow-for-teams-api).
+
+Click on the "Generate token" button.
+
+### Step 2: Store the token in Roadie
+
+Visit `https://<tenant-name>.roadie.so/administration/settings/secrets` and enter the API key value from above into `STACKOVERFLOW_API_KEY`.
+
+### Step 3: Enable private or public Stackoverflow configuration
+
+Visit `https://<tenant-name>.roadie.so/administration/settings/stackoverflow` and select private checkbox for private Stackoverflow Teams. You also need to set up the base API URL for Stackoverflow, which is in most cases, including private Teams integration, `https://api.stackexchange.com/2.2`
```

Comment:
"select the private checkbox"

Reply:
It might be worth splitting the second sentence into multiple sentences. It is quite long and a little confusing.

"You also need to set up the base API URL for Stackoverflow, which is in most cases, including private Teams integration"

Reply:
I split that into few sentences.